db/migrate/20241008145227_create_national_hearing_queue_entries.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class CreateNationalHearingQueueEntries < ActiveRecord::Migration[6.1]
+  def change
+    drop_view :national_hearing_queue_entries, materialized: true
+    create_view :national_hearing_queue_entries, materialized: true
+  end
```

Comment:
```suggestion
  def up
    create_view :national_hearing_queue_entries, materialized: true
  end
  
    def down
      drop_view :national_hearing_queue_entries, materialized: true
    end
  end
```
Having the create and drop in the same `change` block doesn't allow for reverting the migration cleanly.